src/__mocks__/handlers.js

```diff
@@ -0,0 +1,84 @@
+import Data from '../data.js';
+
+export default class Handlers {
+  static removeTask(index, allTasks) {
+    allTasks.splice(index, 1);
+    allTasks.forEach((item, index) => {
+      item.index = index;
+    });
+
+    const existingLists = document.querySelectorAll('.task-item');
+    existingLists.forEach((item) => item.remove());
+
+    const { renderList } = Handlers;
+    renderList(allTasks);
+    return allTasks;
+  }
+
+  static renderList(list) {
+    const clearAll = document.getElementById('clear-all');
+    list.forEach((item, index) => {
+      item.index = index;
+    });
+    const listItems = list.map(
+      (task, index) => `
+            <li class="task-item" title="Double click description to edit"> 
+                <p>
+                    <span>
+                        <label for="task list"> 
+                            <input  class="status"  
+                                    data-index="${index}" 
+                                    type="checkbox"${
+  task.completed ? 'checked' : ''
+} 
+                            />
+                            <span  class="description"> ${
+  task.description
+} </span>
+                        </label>
+                    </span>
+                </p>
+                <p>
+                    <i data-index="${index}" class="remove-item">&times</i>
+                    <span class="drag-around">&#8942</span>
+                </p>
+            </li>
+            `,
+    );
+
+    clearAll?.insertAdjacentHTML('beforebegin', listItems.join(''));
```

Comment:
- [ ] great job on writing the tests 👍 however not all tests are passing as you can see in the below print screen. which is due to this line of code.
![Capture](https://user-images.githubusercontent.com/39240820/145427801-22628c38-88af-4ee8-ab40-c74807b63f15.PNG)

Reply:
Hello Michael Threels. Thank you for making good observations. In the code review, you commented that some tests failed. Although, when I and both my coding partners tested on our PCs, all the tests passed. How can we solve this problem?
![image](https://user-images.githubusercontent.com/45460192/145430259-e15fdaf6-c57b-4e67-b989-552523258ab9.png)

Reply:
For me is everything working, so you are approved 🥳 🎉 

![imagen](https://user-images.githubusercontent.com/60789317/145436869-f96dbadd-bc32-452c-866f-69dc0a28b974.png)